arch/xtensa/src/esp32/Kconfig

```diff
@@ -1423,6 +1389,48 @@ config ESP32_FREERUN
 endmenu # Timer/counter Configuration
 endif # ESP32_TIMER
 
+menu "RTC Configuration"
+	depends on ESP32_RTC
+
+choice ESP32_RTC_CLK_SRC
+	prompt "RTC clock source"
+	default ESP32_RTC_CLK_SRC_INT_RC
+	---help---
+		Choose which clock is used as RTC clock source.
+
+		- "Internal 150kHz oscillator" option provides lowest deep sleep current
+			consumption, and does not require extra external components. However
+			frequency stability with respect to temperature is poor, so time may
+			drift in deep/light sleep modes.
+		- "External 32kHz crystal" provides better frequency stability, at the
+			expense of slightly higher (1uA) deep sleep current consumption.
+		- "External 32kHz oscillator" allows using 32kHz clock generated by an
+			external circuit. In this case, external clock signal must be connected
+			to 32K_XN pin. Amplitude should be <1.2V in case of sine wave signal,
+			and <1V in case of square wave signal. Common mode voltage should be
+			0.1 < Vcm < 0.5Vamp, where Vamp is the signal amplitude.
+			Additionally, 1nF capacitor must be connected between 32K_XP pin and
+			ground. 32K_XP pin can not be used as a GPIO in this case.
+		- "Internal 8.5MHz oscillator divided by 256" option results in higher
+			deep sleep current (by 5uA) but has better frequency stability than
+			the internal 150kHz oscillator. It does not require external components.
+
+	config ESP32_RTC_CLK_SRC_INT_RC
+		bool "Internal 150kHz RC oscillator"
+
+	config ESP32_RTC_CLK_SRC_EXT_XTAL
+		bool "External 32kHz crystal"
+		select ESP_SYSTEM_RTC_EXT_XTAL
+
+	config ESP32_RTC_CLK_SRC_EXT_OSC
+		bool "External 32kHz oscillator at 32K_XN pin"
+
+	config ESP32_RTC_CLK_SRC_INT_8MD256
+		bool "Internal 8.5MHz oscillator, divided by 256 (~33kHz)"
```

Comment:
Please remove leading TABs for this block

Reply:
Fixed, thanks.